code/modules/recycling/sortingmachinery.dm

```diff
@@ -166,62 +170,62 @@
 
 	if(istype(target, /obj/item) && !(istype(target, /obj/item/storage) && !istype(target,/obj/item/storage/box) && !istype(target, /obj/item/shippingPackage)))
 		var/obj/item/O = target
-		if(use(1))
-			var/obj/item/smallDelivery/P = new /obj/item/smallDelivery(get_turf(O.loc))	//Aaannd wrap it up!
-			if(!istype(O.loc, /turf))
-				if(user.client)
-					user.client.screen -= O
-			P.wrapped = O
-			O.loc = P
-			var/i = round(O.w_class)
-			if(i in list(1,2,3,4,5))
-				P.icon_state = "deliverycrate[i]"
-				P.w_class = i
-			P.add_fingerprint(usr)
-			O.add_fingerprint(usr)
-			add_fingerprint(usr)
-		else
-			return
+		if(!use(1))
+			return FALSE
+		var/obj/item/smallDelivery/P = new /obj/item/smallDelivery(get_turf(O.loc)) //Aaannd wrap it up!
+		if(!istype(O.loc, /turf))
+			if(user.client)
+				user.client.screen -= O
+		P.wrapped = O
+		O.loc = P
+		var/i = round(O.w_class)
+		if(i in list(1,2,3,4,5))
+			P.icon_state = "deliverycrate[i]"
+			P.w_class = i
+		P.add_fingerprint(user)
+		O.add_fingerprint(user)
+		add_fingerprint(user)
+
 	else if(istype(target, /obj/structure/closet/crate))
 		var/obj/structure/closet/crate/O = target
 		if(O.opened)
 			return
-		if(amount >= 3 && do_after_once(user, 15, target = target))
-			if(O.opened || !use(3))
-				return
-			var/obj/structure/bigDelivery/P = new /obj/structure/bigDelivery(get_turf(O.loc))
-			P.icon_state = "deliverycrate"
-			P.wrapped = O
-			O.loc = P
-		else
-			to_chat(user, "<span class='notice'>You need more paper.</span>")
-			return
-	else if(istype (target, /obj/structure/closet))
+		if(amount < 3)
+			to_chat(user, "<span class='warning'>You need more paper.</span>")
+			return FALSE
+		if(!do_after_once(user, 15, target = target) || O.opened || !use(3))
```

Comment:
In this case it's in that order so that the check fails if someone's opened the crate when the timer ends, there's an initial `O.opened` check just above on line 191.

Reply:
Makes sense, nevermind.